vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -558,6 +571,18 @@ supported:
           String="Supports [key-as-segment convention](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_KeyasSegmentConvention) for addressing entities within a collection" />
       </Term>
 
+      <Term Name="RelaxedSystemQueryOptionNamesSupported" Type="Core.Tag" Nullable="false" DefaultValue="true"

```

Comment:
Do we need this? I would just say it's required for 4.01, but that clients should use canonical formats for interoperability. I can't imagine a client querying this and then adjusting behavior based on the value.

Reply:
No, we don't need it